src/quality/quality.js

```diff
@@ -196,7 +196,9 @@ Object.assign(MediaElementPlayer.prototype, {
 			qualitiesSelector.style.height = `${qualitiesSelector.querySelector('ul').offsetHeight}px`;
 			qualitiesSelector.style.top = `${(-1 * parseFloat(qualitiesSelector.offsetHeight))}px`;
 			qualityButton.setAttribute('aria-expanded', 'true');
-			qualitiesList.focus();
+			const selectedLabel = qualitiesSelector.querySelector('.' + t.options.classPrefix + 'qualities-selected');
```

Comment:
True. Line 188 `qualityButton.removeAttribute('aria-expanded');` can also be removed. Because it´s a toggle, it doesn´t have to be removed before being set anew.